### ISSUE TITLE
Failed prop type: H1

### ISSUE THREAD

**Issue by alex-kakashi** [GeekyAnts/NativeBase#516]
Hi there,

I want to learning basic NativeBase. Unfortunately the first tag to be included in the Content section was an H1 tag which show warning failed prop type H1.

## react-native, react and native-base version
react: 15.4.2
react-native: 0.41.2
native-base: 2.0.6

## Expected behaviour
Hide log warning H1.

## Actual behaviour
Show log warning H1, H2, H3 every time.

## Steps to reproduce (code snippet or screenshot)
```
         <Content>
            <H1>Header One</H1>
            <H2>Header Two</H2>
            <H3>Header Three</H3>
            <Text>Default</Text>
        </Content>
```

## Screenshot of emulator/device
![simulator-screen-shot-feb-15 -2560-be -10 11 15-am](https://cloud.githubusercontent.com/assets/3070756/22959722/19ae9574-f36b-11e6-82dd-5653ab045d20.png)

## Is the bug present in both ios and android or in any one of them?
I have tried test on simulator iOS.

## Any other additional info which would help us debug the issue quicker.
You can debug with example [typography.](http://nativebase.io/docs/v2.0.0/components#typography)

_Thanks for all your work and help!_

**Comment by sankhadeeproy007** [comment]
Fixed with 2.0.7

**Comment by achmadk** [comment]
In the latest version (2.3.7), I get warning message when using H1 component said `Failed prop type: H1 should have only string`

this is the stack trace
```bash
Warning: Failed prop type: H1 should have only string
     in H1 (at connectStyle.js:384)
     in Styled(H1) (at index.js:10)
     in RCTView (at View.js:71)
     in View (at ScrollView.js:717)
     in RCTScrollView (at ScrollView.js:811)
     in ScrollView (at KeyboardAwareHOC.js:329)
     ...
```

**Comment by akhil-geekyants** [comment]
@achmadk can you check whether you are using any value other than string with H1. Can you paste some code here. 

See docs [here](http://docs.nativebase.io/Components.html#Typography)

**Comment by achmadk** [comment]
Sorry for your inconvenience, Mr. @akhil-geekyants . Accidentally, I am using number value with H1. Maybe you can update Native Base docs then replace option for H1, H2, and H3 component from user-defined to string, or inform that typography components will display `Failed prop type` warning if using any value other than string.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.8](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.8)
